deliver/lib/deliver/runner.rb

```diff
@@ -62,6 +66,19 @@ def upload_metadata
       UploadAssets.new.upload(options) # e.g. app icon
     end
 
+    def prepare_app_icons(options = {})
+      app_icon_path = options[:app_icon] if options[:app_icon]
+      app_icon_metadata = File.join(options[:metadata_path], "app_icon.png")
```

Comment:
Will this crash if `metadata_path` isn't set? Is `metadata_path` optional? It seems to be set in `detect_values.rb`